db/migrate/20150802213507_add_validations.rb

```diff
@@ -0,0 +1,8 @@
+class AddValidations < ActiveRecord::Migration
+  def change
+    change_column :pools, :tournament_id, :integer, :null => false
+    change_column :pool_fighters, :pool_id, :integer, :null => false
+    change_column :pool_fighters, :fighter_id, :integer, :null => false
+    change_column :exchanges, :fighter_id, :integer, :null => false
```

Comment:
What about `MatchFighters` and other such intermediate tables? Are they up to date?